ui/app/components/chart-primitives/h-annotations.hbs

```diff
@@ -0,0 +1,17 @@
+<div data-test-annotations class="line-chart-annotations" style={{this.chartAnnotationsStyle}} ...attributes>
+  {{#each this.processed key=@key as |annotation|}}
+    <div data-test-annotation class="chart-horizontal-annotation" style={{annotation.style}}>
+      <button
+        type="button"
+        title={{annotation.a11yLabel}}
+        class="indicator {{if (or
+          (and @key (eq-by @key annotation.annotation @activeAnnotation))
+          (and (not @key) (eq annotation.annotation @activeAnnotation))
+        ) "is-active"}}"
```

Comment:
Since there’s already a pattern here of calculating annotation things in the Javascript for use in the template (vs needing to extract to another component), maybe this `is-active` calculation could happen there too? But I’ve definitely created some lengthy boolean logic in templates so I don’t insist haha

Reply:
Good call! Especially since all that pre-processing is already happening.